content/docs/tasks/telemetry/distributed-tracing/overview/index.md

```diff
@@ -26,29 +26,32 @@ To do this, an application needs to collect and propagate the following headers
 * `x-b3-flags`
 * `x-ot-span-context`
 
-If you look in the sample services, you can see that the `productpage` service (Python) extracts the required headers from an HTTP request:
+If you look in the sample services, you can see that the `productpage` service (Python) extracts the required headers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:
```

Comment:
The original had some of the issues I fix below.
```suggestion
If you look at the Python `productpage` service, for example, you see that the service extracts the required headers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:
```

Reply:
done.